### PR TITLE
Adds publications info to get_profile_info

### PR DESCRIPTION
**Pull request by purujitgoyal**

### CODE REVIEW

File: openreview/tools.py [hunk 1]
```diff
@@ -1577,6 +1577,7 @@ def get_profile_info(profile):
     emails = set()
     relations = set()
     publications = set()
+    publication_years = 3
```
Comment (melisabok) [inline openreview/tools.py:1580]: do you think the default should be last 3 years? I was thinking to use all the existing publications as the default implementation.

Comment (purujitgoyal) [reply openreview/tools.py:1580]: Depends on how much extra time would it take.

Comment (melisabok) [reply openreview/tools.py:1580]: It shouldn't take too much time to compare the author relationship. Then we can parametrize the last year for publications,  history of domains and relations.

Comment (purujitgoyal) [reply openreview/tools.py]: I have made the default to consider all publications.

**Review by melisabok — APPROVED** [review]

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**